apps/src/music/blockly/blocks/simple2.js

```diff
@@ -80,9 +80,9 @@ export const triggeredAtSimple2 = {
     );
     return `
         var __insideWhenRun = false;
-        ${varName} = MusicPlayer.getPlayheadPosition();
-        currentMeasureLocation = Math.ceil(${varName});
-        ProgramSequencer.playSequentialWithMeasure(currentMeasureLocation);
+        ${varName} = MusicPlayer.getCurrentPlayhead();
+        currentPlayheadLocation = Math.ceil(${varName});
```

Comment:
Curious - do we need the assignment to `${varName}`? Could this just be `currentPlayheadLocation = Math.ceil(MusicPlayer.getCurrentPlayhead())`?

Reply:
Nice catch.  Will reduce even further!